src/core/server/decorators/interaction.ts

```diff
@@ -0,0 +1,28 @@
+import alt, { IVector3 } from 'alt-server';
+import { Athena } from '../api/athena';
+
+export function interaction(
```

Comment:
Provide an example and document this specific new function / decorator with an example.